presto-cassandra/pom.xml

```diff
@@ -23,6 +23,11 @@
             <artifactId>cassandra-driver</artifactId>
```

Comment:
It wasn't moved, but was already there correct? E.g: no new files were added in this commit

Reply:
`RebindSafeMBeanServer` was copied into `presto-plugin-toolkit` and all other copies of `RebindSafeMBeanServer` were removed.